be/src/index.ts

```diff
@@ -17,7 +17,7 @@ const main = async () => {
     entities: [Hello],
   };
   try {
-    let conn = await createConnection({ ...config });
+    await createConnection({ ...config });
```

Comment:
I know this is nitpicking, but maybe just comment the line with the `conn` instead of deleting it, because we'll need it later anyways.